tests/manual-test-cases/Group19-ROBO/19-3-ROBO-VM-Placement.md

```diff
@@ -0,0 +1,36 @@
+Test 19-3 - ROBO - VM Placement
+=======
+
+# Purpose:
+To verify that the VM placement feature specified works as expected in a vSphere ROBO Advanced environment without DRS.
+The current placement strategy is to avoid bad host selection, instead of selecting the "best" possible host.
+
+# References:
+* [vSphere Remote Office and Branch Office](http://www.vmware.com/products/vsphere/remote-office-branch-office.html)
+* [VM Placement without DRS](https://github.com/vmware/vic/issues/7282)
+
+# Environment:
+This test requires access to VMware Nimbus cluster for dynamic ESXi and vCenter creation. This test should be executed in the following topologies and should have vSAN enabled.
+* 1 vCenter host with 3 clusters, where 1 cluster has 1 ESXi host and the other 2 clusters have 3 ESXi hosts each
+* 2 vCenter hosts connected with ELM, where each vCenter host has a cluster/host/datacenter topology that emulates a customer environment (exact topology TBD)
+
+# Test Steps:
+1. Deploy a ROBO Advanced vCenter testbed for both environments above
+2. Install the VIC appliance on vCenter
+3. Create and run several containers in succession
+4. Gather host metrics and check container placements
+5. Delete the containers created in Step 3
+6. Create and run several containers in parallel
+7. Gather host metrics and check container placements
+8. Delete the containers created in Step 5
+9. Run the regression test suite
+10. Delete the VCH
+
+# Expected Outcome:
+* All steps should succeed.
+* In Step 2, the VCH should be placed on a host that satisfies the license and other feature requirements.
+* In Steps 3 and 6, containers shouldn't fail to be created/started unless the cluster resources/limits are exhausted.
+* In Steps 4 and 7, containers should be placed according to the criteria defined in [Purpose](#purpose). More details are TBD.
```

Comment:
I would actually number the references and refer here to the 2nd ref, not the purpose